waltz-test/src/main/python/waltz_ducktape/tests/validation/scalability_test.py

```diff
@@ -97,3 +109,66 @@ def scale_up_replica(self, src_node_idx, added_node_idx, num_active_partitions,
         assert added_node_max_transaction_id == expected_max_transaction_id, \
             "New transactions failed to reach new replica, expected max transaction ID = {}, actual max transaction ID = {}" \
             .format(expected_max_transaction_id, added_node_max_transaction_id)
+
+    def get_number_of_partitions_assigned_to_server(self, server_cli, server):
+        return int(re.search('There are (\d+) partitions for current server', server_cli.list_partition(server)).group(1))
+
+    def get_random_server_node_instance(self, waltz_server_cluster):
```

Comment:
nit: not `waltz_server_cluster` because this parameter only represents the list of waltz server nodes. So, made be just `waltz_server` should work.